public/app/features/explore/state/main.test.ts

```diff
@@ -204,238 +199,15 @@ describe('Explore reducer', () => {
           .givenReducer(exploreReducer, initialState)
           .whenActionIsDispatched(splitCloseAction({ itemId: ExploreId.right }))
           .thenStateShouldEqual(({
-            split: false,
             left: leftItemMock,
-            right: initialExploreState.right,
+            right: undefined,
           } as unknown) as ExploreState);
       });
     });
   });
-
-  describe('when updateLocation is dispatched', () => {
```

Comment:
We don't actually react to updateLocation anymore. Handling of these cases should be tested in Wrapper.test.tsx in a component integration tests.